daphne_worker/src/durable/aggregate_store.rs

```diff
@@ -374,7 +374,8 @@ impl AggregateStore {
                 js_sys::Reflect::set(
                     &chunks_map,
                     &JsValue::from_str(METADATA_KEY),
-                    &serde_wasm_bindgen::to_value(&meta)?,
+                    &serde_wasm_bindgen::to_value(&meta)
+                        .map_err(|e| worker::Error::RustError(e.to_string()))?,
```

Comment:
Is this because `worker` depends on an earlier version of serde-wasm-bindgen? If that's the case it would be better to make a PR to them to bump `serde-wasm-bindgen` first or not bumping the version of `serde-wasm-bindgen` ourselves yet.

I checked what changed and it doesn't seem relevant to us

Reply:
Done.